tasks/rvm.rake

```diff
@@ -1,81 +0,0 @@
-require 'rvm'
```

Comment:
@MSP-Greg : can you please revert this change?
I still used this one when I released the last version of the gem.

I want to do a clean up as well, but not in the scope of this PR.

Reply:
From the current file, which was last updated in Apr-2016...
```ruby
RVM_RUBIES = ['ruby-1.8.6-p398', 'ruby-1.9.1-p243', 'ruby-1.9.2-p136', 'ruby-2.2.3p173']
```

> I still used this one when I released the last version of the gem.

For testing?  That's what CI is for.  BTW, releases, maybe a sensible `required_ruby_version` in the gemspec?  I'd suggest 2.4...

Reply:
@MSP-Greg : that is not for testing.